Autodiscover/AutodiscoverService.cs

```diff
@@ -1519,18 +1522,11 @@ internal HttpRequestMessage PrepareHttpRequestMessageForUrl(Uri url)
 
         internal HttpClient PrepareHttpClient()
         {
-            var httpClientHandler = new HttpClientHandler()
-            {
-                PreAuthenticate = this.PreAuthenticate,

```

Comment:
If you drop some of public props, does it make sense to mark them deprecated?

Reply:
don't understand what do you mean ...

Reply:
If PreAuthenticate is not used on AutodiscoverService level then it shall be dropped/commented out?

Reply:
it is not only PreAuthenticate there are Proxy and UseDefaultCredentials, which are also not used with new implementation, I left them for possible future use.